docs/self-managed/zeebe-deployment/configuration/gateway.md

```diff
@@ -0,0 +1,256 @@
+---
+id: gateway-config
+title: "Gateway configuration"
+sidebar_label: "Gateway configuration"
+description: "Let's analyze how to configure the Zeebe gateway"
+---
+
+A complete gateway configuration template is available in the [Zeebe repo](https://github.com/camunda/zeebe/blob/main/dist/src/main/config/gateway.yaml.template).
```

Comment:
💭 eventually the versioned configuration docs and the content this link will diverge, given main will be 8.3 anytime now. Do we have similar cases, what could be the options to keep them sync? (copy the yaml for a version into the docs? link to the stable branches based on version and only to main from `next`?)

Reply:
Good point. We don't have a process for this currently. Looking at the tags in the Zeebe repo, there are many tag options (8.1.x, for example) per minor. 

Linking to main in `next` seems easiest. But every time we cut a new release, should we point to 8.x.0? We could add a note that users could look for other versions in the repo.

Reply:
For every supported minor release we have a `stable/8.x` branch, e.g. the newest https://github.com/camunda/zeebe/tree/stable/8.2, these could be used.